README.md

```diff
@@ -1,11 +1,12 @@
-# connect-crosstest
+# connect-conformance
```

Comment:
Just `conformance` seems more straight-forward to me.

Reply:
Should this be lowercase `conformance`? Also, on line 9 then should it say 

> `conformance` runs a suite of cross-compatibility...

Kinda sounds weird to me but if you guys are fine with it, works for me.

Reply:
The sentence sounds weird to me too, but couldn't you simply use "This repository runs..." to avoid it?